lib/cc/cli/source_fingerprint.rb

```diff
@@ -0,0 +1,69 @@
+require "digest/md5"
+
+module CC
+  module CLI
+    class SourceFingerprint
+      def initialize(output)
+        @output = output
+      end
+
+      def compute
+        md5 = Digest::MD5.new
+        md5 << key
```

Comment:
That's a good question and I think that we're okay with the new behavior. If source code relevant to an existing issue is changed, we shouldn't consider the existing issue up-to-date. If a new analysis determines that the same kind of issue (identical check_name) is present for the updated source, code, we should emit a fresh issue with a new fingerprint. So in that case, you'd get a fixed issue and a new issue.